src/Label/Label.jsx

```diff
@@ -0,0 +1,104 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PropTypes, PureComponent} from "react";
+
+import {Tooltip} from "../Tooltip";
+
+import "./Label.less";
+
+
+export default class Label extends PureComponent {
+  focus() {
+    if (this.refs.label) {
+      this.refs.label.focus();
+    }
+  }
+
+  render() {
+    const {cssClass} = Label;
+    const {
+      children,
+      className,
+      color,
+      size,
+      tooltip,
+      tooltipPlacement,
+      tooltipTextAlign,
+      ...passthroughProps,
```

Comment:
So `passthroughProps` is hitting the underlying `span` - maybe we should clarify that these are specifically `span` properties, and how do they interact with the existing `tabIndex` and `className` ones?